<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>588</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTextEdit" name="textEdit">
      <property name="styleSheet">
       <string notr="true">border-image: url(:/new/ICOD/images/girl (2).jpg);</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>588</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_F">
    <property name="title">
     <string>文件(&amp;F)</string>
    </property>
    <addaction name="newAction"/>
    <addaction name="openAction"/>
    <addaction name="saveAction"/>
    <addaction name="saveAsAction"/>
    <addaction name="separator"/>
    <addaction name="pintAction"/>
    <addaction name="separator"/>
    <addaction name="exitAction"/>
   </widget>
   <widget class="QMenu" name="menu_E">
    <property name="title">
     <string>编辑(&amp;E)</string>
    </property>
    <addaction name="cancelAction"/>
    <addaction name="cutAction"/>
    <addaction name="copyAction"/>
    <addaction name="pasteAction"/>
    <addaction name="deleteAction"/>
    <addaction name="separator"/>
    <addaction name="seekAction"/>
    <addaction name="findAction"/>
    <addaction name="replaceAction"/>
    <addaction name="gotoAction"/>
    <addaction name="separator"/>
    <addaction name="checkAction"/>
    <addaction name="dataAction"/>
   </widget>
   <widget class="QMenu" name="menu_O">
    <property name="title">
     <string>格式(&amp;O)</string>
    </property>
    <addaction name="autoAction"/>
    <addaction name="fontAction"/>
    <addaction name="colorAction"/>
   </widget>
   <widget class="QMenu" name="menu_V">
    <property name="title">
     <string>查看(&amp;V)</string>
    </property>
    <addaction name="statusAction"/>
   </widget>
   <widget class="QMenu" name="menu_H">
    <property name="title">
     <string>帮助(&amp;H)</string>
    </property>
    <addaction name="helpAction"/>
    <addaction name="aboutAction"/>
   </widget>
   <addaction name="menu_F"/>
   <addaction name="menu_E"/>
   <addaction name="menu_O"/>
   <addaction name="menu_V"/>
   <addaction name="menu_H"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>true</bool>
   </attribute>
   <addaction name="newAction"/>
   <addaction name="openAction"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="newAction">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/new/ICOD/images/girl (2).jpg</normaloff>:/new/ICOD/images/girl (2).jpg</iconset>
   </property>
   <property name="text">
    <string>新建(&amp;N)</string>
   </property>
  </action>
  <action name="openAction">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/new/ICOD/images/girl (1).jpg</normaloff>:/new/ICOD/images/girl (1).jpg</iconset>
   </property>
   <property name="text">
    <string>打开(&amp;O)</string>
   </property>
  </action>
  <action name="saveAction">
   <property name="text">
    <string>保存(&amp;S)</string>
   </property>
  </action>
  <action name="saveAsAction">
   <property name="text">
    <string>另存为(&amp;A)</string>
   </property>
  </action>
  <action name="action_P">
   <property name="text">
    <string>打印(P)</string>
   </property>
  </action>
  <action name="pintAction">
   <property name="text">
    <string>打印(&amp;P)</string>
   </property>
  </action>
  <action name="action_Q">
   <property name="text">
    <string>退出(Q)</string>
   </property>
  </action>
  <action name="cancelAction">
   <property name="text">
    <string>撤销(&amp;U)</string>
   </property>
  </action>
  <action name="cutAction">
   <property name="text">
    <string>剪切(&amp;T)</string>
   </property>
  </action>
  <action name="copyAction">
   <property name="text">
    <string>复制(&amp;C)</string>
   </property>
  </action>
  <action name="pasteAction">
   <property name="text">
    <string>粘贴(&amp;P)</string>
   </property>
  </action>
  <action name="deleteAction">
   <property name="text">
    <string>删除(&amp;L)</string>
   </property>
  </action>
  <action name="seekAction">
   <property name="text">
    <string>查找(&amp;F)</string>
   </property>
  </action>
  <action name="findAction">
   <property name="text">
    <string>查找下一个(&amp;N)</string>
   </property>
  </action>
  <action name="replaceAction">
   <property name="text">
    <string>替换(&amp;R)</string>
   </property>
  </action>
  <action name="gotoAction">
   <property name="text">
    <string>转到(&amp;G)</string>
   </property>
  </action>
  <action name="checkAction">
   <property name="text">
    <string>全选(&amp;A)</string>
   </property>
  </action>
  <action name="dataAction">
   <property name="text">
    <string>时间/日期(&amp;D)</string>
   </property>
  </action>
  <action name="exitAction">
   <property name="text">
    <string>退出(&amp;Q)</string>
   </property>
  </action>
  <action name="autoAction">
   <property name="text">
    <string>自动换行(&amp;W)</string>
   </property>
  </action>
  <action name="fontAction">
   <property name="text">
    <string>字体(&amp;F)...</string>
   </property>
  </action>
  <action name="statusAction">
   <property name="text">
    <string>状态栏(&amp;S)</string>
   </property>
  </action>
  <action name="helpAction">
   <property name="text">
    <string>查看帮助(&amp;H)</string>
   </property>
  </action>
  <action name="aboutAction">
   <property name="text">
    <string>关于记事本(&amp;A)</string>
   </property>
  </action>
  <action name="colorAction">
   <property name="text">
    <string>颜色(&amp;c)</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
